myproject/app/api/user.py

```diff
@@ -60,18 +62,18 @@ def update_user():
     user_id = data.get('id')
```

Comment:
can you explain the purpose of user_id? How is the API restricted to only logged-in users?